operator/pkg/helmreconciler/reconciler.go

```diff
@@ -216,15 +229,21 @@ func (h *HelmReconciler) processRecursive(manifests name.ManifestMap) *v1alpha1.
 func (h *HelmReconciler) Delete() error {
 	iop := h.iop
 	if iop.Spec.Revision == "" {
-		return h.Prune(nil, true)
+		err := h.Prune(nil, true)
+		h.reportOwnedObjectCountMetrics()
+		h.reportPrunedObjectKind()
```

Comment:
use a defer func for this maybe? seems we always want to call on return

Reply:
Yeah. Looking at it, it makes sense. Done.